README.md

```diff
@@ -41,9 +41,10 @@ validation includes invalidly formatted JSON files and missing
 mandatory fields. There will be a checklist that you are expected 
 to complete and attest to its accuracy. Once you submit your request, 
 your pull request will be reviewed by the registry editors. Changes 
-regarding the required criteria may be requested. If there are no 
-objections or changes requested, your DID method will be 
-registered after a minimum of 7 days and a maximum of 30 days.
+regarding the required criteria may be requested. If there are at 
+least two reviews by registry maintainers, and no objections or 
```

Comment:
are we sure 2 is enough, I suggest we make it 100%... in the spirit of W3C Consensus.

Reply:
Our history is showing that only two of the CODEOWNERS are regularly maintaining this repository... so, raising the bar higher will almost guarantee that no more entries go in, which will make the current maintainer's lives more difficult.

Reply:
If folks are not performing their editorial duties, it's better to remove them, so we have room for more engaged editors.

I prefer code owners to be used and for the review to be enforced by the version control system and branch merge rules.

Reply:
> I prefer code owners to be used and for the review to be enforced by the version control system and branch merge rules.

Agree that merges should be enforced by branch merge rules... we can set that up as soon as we figure out what we're doing here.

I'm not sure reducing CODEOWNERS is a good move at this time... people go through periods of their work life where they might be very busy for months at a time, or when we need more input from folks we can call on the other maintainers.

Reply:
https://docs.github.com/en/enterprise-cloud@latest/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners

> When someone with admin or owner permissions has enabled required reviews, they also can optionally require approval from a code owner before the author can merge a pull request in the repository.

Reply:
@OR13 I need you to resolve this conversation in order to merge.

Reply:
I am resolving this conversation, since text was added to address it here:

https://github.com/w3c/did-spec-registries/pull/489/commits/f2a48d9bb942a7509507c9311cab444d7e8c488d